### PR TITLE
Readyyy!!!

### PR DESCRIPTION
**Pull request by kyranet**
### Proposed Semver Increment Bump: [MAJOR/MINOR/PATCH]
minor
### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)

- Hey owner, I'm ready!

### (If Applicable) What Issue does it fix?
 Fixes...

### CODE REVIEW

File: classes/client.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ module.exports = class Komada extends Discord.Client {
     }));
     await this.configuration.initialize(this);
     this.ready = true;
+    this.emit("log", `Ready to serve ${this.guilds.size} guilds.`);
```
Comment (CyberiumShadow) [inline classes/client.js]: Needs to be more fancy. 1000x fancier for Komada.

Failure to do so will be 1000 years in the slow lane, no appeal.

**Review by CyberiumShadow — CHANGES_REQUESTED** [review]

File: classes/client.js (hunk 1 above)
Comment (kyranet) [reply classes/client.js]: I have an idea, let's attach this log with a rocket.

Comment (kyranet) [reply classes/client.js]: The heck, why not customizable with `komada.configs.readyMessage`?

Comment (CyberiumShadow) [reply classes/client.js]: Do eet.

**Review by CyberiumShadow — APPROVED** [review]

**Comment by CyberiumShadow** [comment]
Komada Ready for Combat!

**Pull request merged**